tsuru/client/init.go

```diff
@@ -0,0 +1,160 @@
+package client
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"strings"
+
+	"github.com/tsuru/tsuru/cmd"
+)
+
+type Init struct{}
+
+func (i *Init) Info() *cmd.Info {
+	return &cmd.Info{
+		Name:  "init",
+		Usage: "init",
+		Desc: `
+Creates a standard example of .tsuruignore , tsuru.yaml and Procfile 
+	on the current project directory.
+
+"Procfile" describes the components required to run an application. 
+	It is the way to tell tsuru how to run your applications;
+
+".tsuruignore" describes to tsuru what it should not add into your 
+	deploy process, via "tsuru app-deploy" command. You can use 
+	".tsuruignore" to avoid sending files that were committed, 
+	but aren't necessary for running the app, like tests and 
+	documentation;
+
+"tsuru.yaml" describes certain aspects of your app, like information
+	about deployment hooks and deployment time health checks.`,
+	}
+}
+
+func (i *Init) Run(context *cmd.Context, client *cmd.Client) error {
+	err := createInitFiles()
+	if err != nil {
+		return err
+	}
+	const msg = `
+Initialized Tsuru sample files: "Procfile", ".tsuruignore" and "tsuru.yaml", 
+for more info please refer to "tsuru init -h" or the docs at docs.tsuru.io`
+	context.Stdout.Write([]byte(msg))
+	err = writeTsuruYaml()
+	if err != nil {
+		return err
+	}
+	err = writeProcfile()
+	if err != nil {
+		return err
+	}
+	return copyGitIgnore()
+}
+
+func createInitFiles() error {
+	wd, err := os.Getwd()
+	if err != nil {
+		return err
+	}
+	fi, err := os.Open(wd)
+	if err != nil {
+		return err
+	}
+	defer fi.Close()
+	dirFiles, err := fi.Readdir(0)
+	if err != nil {
+		return err
+	}
+	initFiles := map[string]string{
+		".tsuruignore": "",
+		"Procfile":     "",
+		"tsuru.yaml":   "",
+	}
+	for _, f := range dirFiles {
+		if _, ok := initFiles[f.Name()]; ok {
+			delete(initFiles, f.Name())
+		}
+	}
+	for f := range initFiles {
+		_, errC := os.Create(f)
+		if errC != nil {
+			return errC
+		}
+	}
+	return nil
+}
+
+func copyGitIgnore() (err error) {
+	in, err := os.Open(".gitignore")
+	if err != nil {
+		dotGit := []byte(".git\n.gitignore\n")
+		return ioutil.WriteFile(".tsuruignore", dotGit, 0644)
+	}
+	defer in.Close()
+	out, err := os.OpenFile(".tsuruignore", os.O_APPEND|os.O_WRONLY, os.ModeAppend)
+	if err != nil {
+		return
+	}
+	defer out.Close()
+	_, err = io.Copy(out, in)
+	if err != nil {
+		return
+	}
+	_, err = out.WriteString("\n.git\n.gitignore\n")
+	return
+}
+
+func writeTsuruYaml() error {
+	const yamlSample = `
```

Comment:
Maybe all lines of the sample should be commented out, or just shell scripts that use "echo" to print something.
The hooks use at the moment will probably fail for most of the applications.